csunplugged/topics/content/en/unplugged-programming/programming-challenges/catch-the-mouse/catch-the-mouse.md

```diff
@@ -0,0 +1,25 @@
+# Catch the mouse
+
+## Requirement:
+
+Remix the program provided at https://scratch.mit.edu/projects/164749053/ and change its name to: **KidBots challenge 2.1 Catch the mouse.**
+
+Give instructions to the “cat” to catch the “mouse” (avoiding the blue tiles) by altering the blocks under the `scratch:when I receive [go v]` block inside the “cat” sprite. You may only use the following custom blocks:
+
+  -   `scratch:move_forward()`
+  -   `scratch:turn_right`
+  -   `scratch:turn_left`
+
+which are the custom blocks found under the "More Blocks" script.
+
+  -   When your instructions are complete, press the “Go” button.
+  -   To move the "cat" to the starting point (x: 200 y:-100), press the "Restart" button .
+  -   To change to a new background press the "Change background" button. We have provided 4 different backdrops for you to try out.
+
+## Testing examples:
+
+You will need to add the correct set of blocks under the `scratch:when I receive [go v]` block so the cat can catch the mouse.
+
+Click on the green flag and press the “Go” button to see the expected output of your program when the correct set of instructions are provided.
```

Comment:
These programs really don't suit the default challenge layout, so may need to revisit this in the future.